src/app/shared/utils/utils.ts

```diff
@@ -74,3 +74,14 @@ export const getParentFolder = (currentFolder: string): string => {
         return index === 0 ? '/' : index > 0 && currentFolder.substr(0, index);
     }
 };
+
+export const isMicrosoftInternalDomain = (): boolean => {
+    const msftInternalDomains = ['redmond.corp.microsoft.com', 'northamerica.corp.microsoft.com', 'fareast.corp.microsoft.com', 'ntdev.corp.microsoft.com', 'wingroup.corp.microsoft.com', 'southpacific.corp.microsoft.com', 'wingroup.windeploy.ntdev.microsoft.com', 'ddnet.microsoft.com', 'europe.corp.microsoft.com'];
```

Comment:
intersting to know if it's a complete list, or if we want to do partial string compariosn. For example, to compare if the domain ends with corp.microsoft.com

Reply:
This is how the vscode source code checks for internal users, not sure if we need to get more exact than that

Reply:
That's is good enough then. Approved.